### PR TITLE
controller-model split for HistoryPane

### PR DESCRIPTION
**Pull request by hendrikebbers**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by hendrikebbers** [comment]
https://github.com/hashgraph/hedera-transaction-tool-demo/pull/432 should be merged before reviewing this PR.

File: tools-ui/src/main/java/com/hedera/hashgraph/client/ui/MainController.java [hunk 1]
```diff
@@ -126,9 +126,9 @@
 import static org.zeroturnaround.zip.commons.FileUtils.copyDirectory;
 import static org.zeroturnaround.zip.commons.FileUtils.deleteDirectory;
 
-public class Controller implements Initializable, GenericFileReadWriteAware {
+public class MainController implements Initializable, GenericFileReadWriteAware {
```
Comment (hendrikebbers) [inline tools-ui/src/main/java/com/hedera/hashgraph/client/ui/MainController.java:129]: somehow this is disarranged. In main branch `MainController` is already used. Based on that it should not be displayed as a change in the PR

Comment (jbair06) [reply tools-ui/src/main/java/com/hedera/hashgraph/client/ui/MainController.java]: Seems that a squash merge or perhaps rebasing can cause this. After changing the target of the merge to another branch and back to main, the issue seems to be gone.

**Review by jbair06 — APPROVED** [review]
Looks like the right direction, will need to do something similar to the rest of the controllers/views.

**Pull request merged**